### PR TITLE
Preserve aspect when stroking SVG.

### LINKED ISSUE #976: ImageFillOriginal does not preserve stroke aspect ratio
1. Drag window to resize only width
2. See vertical and horizontal lines are no longer the same width

```go
package main

import (
	"fyne.io/fyne"
	"fyne.io/fyne/app"
	"fyne.io/fyne/canvas"
	"fyne.io/fyne/layout"
)

func main() {
	a := app.New()
	w := a.NewWindow("Lab")
	w.SetMaster()
	w.Resize(fyne.NewSize(400, 300))
	img := &canvas.Image{
		Resource: &fyne.StaticResource{
			StaticName:    "image.svg",
			StaticContent: []byte{60, 33, 68, 79, 67, 84, 89, 80, 69, 32, 115, 118, 103, 32, 80, 85, 66, 76, 73, 67, 32, 34, 45, 47, 47, 87, 51, 67, 47, 47, 68, 84, 68, 32, 83, 86, 71, 32, 49, 46, 49, 47, 47, 69, 78, 34, 32, 34, 104, 116, 116, 112, 58, 47, 47, 119, 119, 119, 46, 119, 51, 46, 111, 114, 103, 47, 71, 114, 97, 112, 104, 105, 99, 115, 47, 83, 86, 71, 47, 49, 46, 49, 47, 68, 84, 68, 47, 115, 118, 103, 49, 49, 46, 100, 116, 100, 34, 62, 10, 60, 115, 118, 103, 32, 118, 101, 114, 115, 105, 111, 110, 61, 34, 49, 46, 49, 34, 32, 120, 109, 108, 110, 115, 61, 34, 104, 116, 116, 112, 58, 47, 47, 119, 119, 119, 46, 119, 51, 46, 111, 114, 103, 47, 50, 48, 48, 48, 47, 115, 118, 103, 34, 32, 120, 109, 108, 110, 115, 58, 120, 108, 105, 110, 107, 61, 34, 104, 116, 116, 112, 58, 47, 47, 119, 119, 119, 46, 119, 51, 46, 111, 114, 103, 47, 49, 57, 57, 57, 47, 120, 108, 105, 110, 107, 34, 32, 120, 109, 108, 58, 115, 112, 97, 99, 101, 61, 34, 112, 114, 101, 115, 101, 114, 118, 101, 34, 32, 119, 105, 100, 116, 104, 61, 34, 52, 56, 48, 34, 32, 104, 101, 105, 103, 104, 116, 61, 34, 50, 52, 48, 34, 32, 118, 105, 101, 119, 66, 111, 120, 61, 34, 48, 32, 48, 32, 52, 56, 48, 32, 50, 52, 48, 34, 62, 10, 32, 32, 32, 32, 60, 114, 101, 99, 116, 32, 120, 61, 34, 48, 34, 32, 121, 61, 34, 48, 34, 32, 119, 105, 100, 116, 104, 61, 34, 52, 56, 48, 34, 32, 104, 101, 105, 103, 104, 116, 61, 34, 50, 52, 48, 34, 32, 102, 105, 108, 108, 61, 34, 119, 104, 105, 116, 101, 34, 32, 47, 62, 10, 32, 32, 32, 32, 60, 112, 97, 116, 104, 32, 102, 105, 108, 108, 61, 34, 110, 111, 110, 101, 34, 32, 115, 116, 114, 111, 107, 101, 61, 34, 98, 108, 97, 99, 107, 34, 32, 115, 116, 114, 111, 107, 101, 45, 108, 105, 110, 101, 99, 97, 112, 61, 34, 114, 111, 117, 110, 100, 34, 32, 115, 116, 114, 111, 107, 101, 45, 108, 105, 110, 101, 106, 111, 105, 110, 61, 34, 114, 111, 117, 110, 100, 34, 32, 115, 116, 114, 111, 107, 101, 45, 119, 105, 100, 116, 104, 61, 34, 49, 52, 34, 32, 100, 61, 34, 77, 32, 49, 48, 48, 44, 54, 48, 32, 76, 32, 49, 48, 48, 44, 49, 56, 48, 32, 76, 32, 49, 54, 48, 44, 49, 56, 48, 34, 32, 47, 62, 10, 60, 47, 115, 118, 103, 62, 10}},
		FillMode: canvas.ImageFillOriginal,
	}
	w.SetContent(fyne.NewContainerWithLayout(
		layout.NewBorderLayout(img, nil, nil, nil),
		img))
	w.CenterOnScreen()
	w.ShowAndRun()
}
```

### PR DESCRIPTION
**Pull request by andydotxyz**
Pretty straight forward - the SVG stroker didn't have the right aspect.
It very subtly changed some of the test images. Double check them all :)

Fixes #976

### Checklist:

- [x] Tests included
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

### CODE REVIEW

**Comment by andydotxyz** [comment]
I figured out that I could test this using the supported ImageFillContain as they both play wtih aspect, so tests now included

File: internal/painter/image.go [hunk 1]
```diff
@@ -49,17 +49,27 @@ func PaintImage(img *canvas.Image, c fyne.Canvas, width, height int) image.Image
...
+				viewAspect := float32(width) / float32(height)
+
+				ww, hh := width, height
+				if viewAspect > aspects[img.Resource] {
+					ww = int(float32(height) * aspect)
+				} else if viewAspect < aspects[img.Resource] {
+					hh = int(float32(width) / aspect)
+				}
+
+				icon.SetTarget(0, 0, float64(ww), float64(hh))
```
Comment (toaster) [inline internal/painter/image.go:64]: There are three pairs of variables for width and height in here. It isn’t obvious what the difference between `width`, `w` and `ww` is.
I suggest to rename `w` and `h` to `sourceWidth` and `sourceHeight` (or `srcW` and `srcH`) and the `ww` and `hh` to `targetWidth` and `targetHeight` or `tgtW` and `tgtH`.

Comment (toaster) [reply internal/painter/image.go:64]: Or maybe `original` (`orig`) and `texture` (`tex`) instead of `source` and `target` if this fits better.

Comment (andydotxyz) [reply internal/painter/image.go:64]: Yeah, I should have fixed this before pushing - thanks

**Comment by andydotxyz** [comment]
Come on travis/GitHub! it is all green

**Pull request merged**